src/tilesource.js

```diff
@@ -569,13 +569,13 @@ $.TileSource.prototype = {
     },
 
     /**
-     * Responsible determining if a the particular TileSource supports the
+     * Responsible determining if the particular TileSource supports the
```

Comment:
While you're in here, it seems like this should be "responsible for determining".